libs/designsystem/src/lib/components/page/page-footer/page-footer.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, ElementRef, HostBinding, Input, OnDestroy } from '@angular/core';
+
+import { PageComponent } from '../page.component';
+
+@Component({
+  selector: 'kirby-page-footer',
+  templateUrl: './page-footer.component.html',
+  styleUrls: ['./page-footer.component.scss'],
+})
+export class PageFooterComponent implements OnDestroy {
+  @Input()
+  @HostBinding('class.padding')
+  hasPadding: boolean = true;
+
+  constructor(private pageComponent: PageComponent, private host: ElementRef<HTMLElement>) {}
```

Comment:
Consider adding `@Optional` and `@SkipSelf`
```suggestion
  constructor(@Optional() @SkipSelf() private pageComponent: PageComponent, private host: ElementRef<HTMLElement>) {}
```